### PR TITLE
Change design and add new parameters

### PR DESCRIPTION
**Pull request by MihaelaDraga**
The cards from main page.tsx were added in separate files, for each card.
Also new parameters were added . For each parameter was added a radio button to turn on and off , also a modal to change the input, and a computation function to make an average.

### CODE REVIEW

File: src/front_end/carsdata/app/components/animations/belt.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import "./style.css"
```
Comment (Elisei-26) [inline src/front_end/carsdata/app/components/animations/belt.js:1]: add ; at the and of all import rows in all folders

File: src/front_end/carsdata/app/components/doorsComponents/BeltCard.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+'use client'
+import React, { useState } from 'react'
+import Belt from '../animations/belt.js'
+
+const BeltCard = (props: any) => {
+    const [belt_state, setstate] = useState(props.belt);
+    const change_belt_status = () => {
+
+          if (belt_state == 'On') {
+                setstate('Off')
```
Comment (Elisei-26) [inline src/front_end/carsdata/app/components/doorsComponents/BeltCard.tsx:10]: ; at the end of row

File: src/front_end/carsdata/app/components/doorsComponents/CigaretteVoltageLighterCard.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+import React from 'react'
+import Cigarette_light from '../animations/cigarette_light.js'
+import { SetStateAction, useState } from 'react';
+import Modal from "./Modal"
+
+const CigaretteVoltageLighter = (props: any) => {
+    const [inputValue, setInputValue] = useState(props.cigarette_lighter);
+    const [randomNumbers, setRandomNumbers] = useState<number[]>([]);
+    const [averageValue, setAverageValue] = useState<number | null>(null);
+    // const [cigaretteLighterVoltage, setCigaretteLighterVoltage] = useState<number | null>(null);
```
Comment (Elisei-26) [inline src/front_end/carsdata/app/components/doorsComponents/CigaretteVoltageLighterCard.tsx:11]: delete unused variables and commented code

File: src/front_end/carsdata/app/components/doorsComponents/DoorPositionCard.tsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
+'use client'
+import React from 'react'
+import Car from '../animations/car.js'
+import { SetStateAction, useState } from 'react';
+import Modal from "./Modal"
+
+const DoorPosition = (props: any) => {
+  const [inputValue, setInputValue] = useState(props.doorparam);
+
+  const handleInputChange = (value: any, id: string) => {
```
Comment (Elisei-26) [inline src/front_end/carsdata/app/components/doorsComponents/DoorPositionCard.tsx:10]: delete id parameter, it is not used

File: src/front_end/carsdata/app/components/doorsComponents/DoorStateCard.tsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
+'use client'
+import React, { useState } from 'react'
+import LockUnlock from '../animations/lock.js'
+
+const DoorState = (props: any) => {
+    const [state, setstate] = useState(props.door_state);
+    const change_status = () => {
+
+          if (state == 'Lock') {
+                setstate('Unlock')
```
Comment (Elisei-26) [inline src/front_end/carsdata/app/components/doorsComponents/DoorStateCard.tsx:10]: ; at the end of row

File: src/front_end/carsdata/app/components/doorsComponents/DoorStatusCard.tsx [hunk 6]
```diff
@@ -0,0 +1,29 @@
+'use client'
+import React, { useState } from 'react'
+import Car from '../animations/car.js'
+
+const DoorStatus = (props: any) => {
+    const [status, setstatus] = useState(props.doorstatus);
+    const change_status = () => {
+
+          if (status == 'open') {
+                setstatus('close')
```
Comment (Elisei-26) [inline src/front_end/carsdata/app/components/doorsComponents/DoorStatusCard.tsx:10]: ; at the end of row

File: src/front_end/carsdata/app/components/doorsComponents/LightsCard.tsx [hunk 7]
```diff
@@ -0,0 +1,29 @@
+'use client'
+import React, { useState } from 'react'
+import Indoor_light from '../animations/indoor_light.js'
+
+const Lights = (props: any) => {
+    const [light_state, setstate] = useState(props.lights);
+    const change_status = () => {
+
+          if (light_state == 'On') {
+                setstate('Off')
```
Comment (Elisei-26) [inline src/front_end/carsdata/app/components/doorsComponents/LightsCard.tsx:10]: ; at the end of row

File: src/front_end/carsdata/app/components/doorsComponents/Modal.tsx [hunk 8]
```diff
@@ -0,0 +1,34 @@
+
+import React, { useState } from 'react';
+
+const Modal = (props: any) => {
+    const [input, setInput] = useState('');
+
+    const handleChange = (event: any) => {
+        props.handleInputChange(event.target.value, props.id);
```
Comment (Elisei-26) [inline src/front_end/carsdata/app/components/doorsComponents/Modal.tsx:8]: delete props.id, it is not used

File: src/front_end/carsdata/app/components/doorsComponents/SerialnumberCard.tsx [hunk 9]
```diff
@@ -0,0 +1,33 @@
+'use client'
+import React from 'react'
+import SerialNumber from '../animations/serialNumber.js'
+import { SetStateAction, useState } from 'react';
+import Modal from "./Modal"
+
+const Serialnumber = (props: any) => {
+    const [inputValue, setInputValue] = useState(props.serialNumber);
+
+    const handleInputChange = (value: any, id: string) => {
```
Comment (Elisei-26) [inline src/front_end/carsdata/app/components/doorsComponents/SerialnumberCard.tsx:10]: id is not used

File: src/front_end/carsdata/app/components/doorsComponents/WindowStatusCard.tsx [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+'use client'
+import React, { useState } from 'react'
+import Window from '../animations/window.js'
+
+const WindowStatusCard = (props: any) => {
+    const [status, setstatus] = useState(props.windows_status);
+    const change_status = () => {
+
+          if (status == 'Closed') {
+                setstatus('Open')
```
Comment (Elisei-26) [inline src/front_end/carsdata/app/components/doorsComponents/WindowStatusCard.tsx:11]: ; at the end of row

**Review by Elisei-26 — CHANGES_REQUESTED** [review]

File: src/front_end/carsdata/app/components/animations/belt.js (hunk 1 above)
Comment (MihaelaDraga) [reply src/front_end/carsdata/app/components/animations/belt.js]: Done

File: src/front_end/carsdata/app/components/doorsComponents/CigaretteVoltageLighterCard.tsx (hunk 3 above)
Comment (MihaelaDraga) [reply src/front_end/carsdata/app/components/doorsComponents/CigaretteVoltageLighterCard.tsx]: Done

File: src/front_end/carsdata/app/components/doorsComponents/DoorStateCard.tsx (hunk 5 above)
Comment (MihaelaDraga) [reply src/front_end/carsdata/app/components/doorsComponents/DoorStateCard.tsx]: Done

File: src/front_end/carsdata/app/components/doorsComponents/DoorStatusCard.tsx (hunk 6 above)
Comment (MihaelaDraga) [reply src/front_end/carsdata/app/components/doorsComponents/DoorStatusCard.tsx]: Done

File: src/front_end/carsdata/app/components/doorsComponents/Modal.tsx (hunk 8 above)
Comment (MihaelaDraga) [reply src/front_end/carsdata/app/components/doorsComponents/Modal.tsx]: Done

File: src/front_end/carsdata/app/components/doorsComponents/WindowStatusCard.tsx (hunk 10 above)
Comment (MihaelaDraga) [reply src/front_end/carsdata/app/components/doorsComponents/WindowStatusCard.tsx:11]: Done

File: src/front_end/carsdata/app/components/doorsComponents/BeltCard.tsx [hunk 11]
```diff
@@ -1,13 +1,13 @@
 'use client'
```
Comment (Elisei-26) [inline src/front_end/carsdata/app/components/doorsComponents/BeltCard.tsx:1]: ; to 'use client' rows too

**Review by Elisei-26 — CHANGES_REQUESTED** [review]

**Review by Elisei-26 — APPROVED** [review]

**Comment by GeorgeCimpoeru** [comment]
Why do we have Battery components inside DOORS components directory ?

**Comment by MihaelaDraga** [comment]
We don't need the Battery, now is deleted

**Review by GeorgeCimpoeru — APPROVED** [review]

**Pull request merged**